lua/bufferline/utils.lua

```diff
@@ -4,14 +4,16 @@
 --
 
 local vim = vim
-local api = vim.api
+local nvim = require'bufferline.nvim'
 local bufname = vim.fn.bufname
 local fnamemodify = vim.fn.fnamemodify
 local split = vim.split
 local join = table.concat
 
+local TYPE_STRING = 'string'
+
 local function len(value)
-  return #value
+  return type(value) == TYPE_STRING and nvim.strwidth(value) or #value
```

Comment:
I don't like this change, `utils.len` is too generic and `strwidth` is a different operation than `len`.